index.js

```diff
@@ -12,13 +16,13 @@ module.exports = (embark) => {
       const [, contractNames] = cmd.split(' ');
 
       if (contractNames) {
-        embark.logger.info('Going to flatten', contractNames);
```

Comment:
line 11: ; // You can use `split` for commands that receive parameters
Is it a leftover from the docs?

Reply:
Haha, yes oops.